src/plugins/missing/README.md

```diff
@@ -9,16 +9,8 @@
 - infos/metadata =
 - infos/description = A plugin which informs the user about a missing backend.
 
-## Introduction
+<!-- TODO [new_backend]: Use this to let kdbOpen run when a plugin is missing -->
```

Comment:
Can you please create an issue? We should get everything important to the issue tracker so that we know when we are finished and can release.

Reply:
#4608